### PR TITLE
Update Introduction by adding Background subsection

### PR DESCRIPTION
**Pull request by bruce-usab**
Update Introduction section, adding sub section Background which characterizes previous 2013 version of WCAG2ICT.

### CODE REVIEW

**Comment by bruce-usab** [comment]
Closes #20

**Comment by bruce-usab** [comment]
I made some minor edits to prose as proposed in #20 
- Reduce some redundancy in first two sentences
- Avoid past tense if possible
- Use <q>Working Group Note</q> instead of <q>note</q>
- Use <q>WCAG</q> instead of <q>WCAG 2.0</q> or <q>WCAG 2.2</q> where possible

File: introduction.md [hunk 1]
```diff
@@ -1,14 +1,18 @@
-Introduction
-------------
+## Introduction
+
 ### Background
 
-<p class="ednote">This section is to be developed by the WCAG2ICT Task Force.</p>
+A W3C [Working Group Note](http://www.w3.org/2005/10/Process-20051014/tr#WGNote) on the topic of applying WCAG 2.0 to non-web content was approved in September of 2013.  That Working Group Note, [Guidance on Applying WCAG 2.0 to Non-Web Information and Communications Technologies (WCAG2ICT)](https://www.w3.org/TR/2013/NOTE-wcag2ict-20130905/) describes how WCAG 2.0 can be applied to ICT, specifically to non-web documents and software.  WCAG21CT is organized by sections to mirror those found in WCAG: Perceivable, Operable, Understandable, and Robust.  WCAG2ICT clarifies when and how WCAG success criteria should be applied to non-web documents and software.  Some success criteria are applicable without modification, and some success criteria are applicable with edits and/or as noted.  Glossary terms were also reviewed for applicability as written.  Level AAA Success Criteria were not addressed in the 2013 WCAG2ICT Working Group Note.
```
Comment (maryjom) [inline introduction.md:5]: Suggest replacing "with edits and/or as noted" with "with edits and/or notes".

File: introduction.md [hunk 2]
```diff
@@ -1,14 +1,18 @@
...
+This document provides informative guidance (guidance that is not normative, and that does not set requirements) with regard to the interpretation and application of [Web Content Accessibility Guidelines (WCAG) 2.2](http://www.w3.org/TR/WCAG22/) \[\[WCAG22\]\] to non-web information and communications technologies (ICT). This document is an update to the [2013 WCAG2ICT Working Group Note](http://www.w3.org/2005/10/Process-20051014/tr#WGNote) (this is in contrast to WCAG 2.1 and WCAG 2.2, which are W3C Recomendations).  Specifically, this document provides informative guidance on applying WCAG 2.2 success criteria to non-web ICT, specifically to non-web documents and software.
```
Comment (maryjom) [inline introduction.md:13]: Editorial - "Recomendations" should be "Recommendations"

Comment (maryjom) [inline introduction.md:13]: Editorial: Misspelled Recommendations.

File: introduction.md (hunk 1 above)
Comment (maryjom) [reply introduction.md]: @bruce-usab I've incorporated these and the changes suggested by Olivia into your branch.

**Pull request merged**